qiskit/transpiler/optimization_metric.py

```diff
@@ -0,0 +1,34 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2025.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+
+"""
+Enumeration of optimization metrics.
+"""
+
+import enum
+
+
+class OptimizationMetric(enum.Enum):
+    """Optimization metric considered during transpilation.
+
+    The metric :data:`COUNT_2Q` targets optimizing the two-qubit gate count of
+    the output circuit.
```

Comment:
```suggestion
    the output circuit. This is generally the preferred choice for
    near-term execution.
```

Reply:
Added in [1a91fee](https://github.com/Qiskit/qiskit/pull/14902/commits/1a91fee5efae8d8b02368215ab72aa2f1e105827).